vstruct/defs/constants/elf.py

```diff
@@ -0,0 +1,905 @@
+EI_NIDENT = 4
```

Comment:
can you add a header, perhaps with a note saying why EI_NIDENT doesn't match sys/elf.h?
```suggestion
## ELF IDENT index constants
# EI_NIDENT differs from sys/elf.h because we only use this to indicate the \x7fELF, but break out the individual
# fields (class/data/fileversion/osabi/abiversion/padding) in the Elf Header vstruct, instead of making them just
# part of the char array.
EI_NIDENT = 4
```